.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,39 @@
+name: Dependabot update node-modules
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    # 22.04's podman has issues with piping and causes tar errors
+    runs-on: ubuntu-20.04
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Run npm-update bot
+        run: |
+          make tools/node-modules
+          git config --global user.name "GitHub Workflow"
+          git config --global user.email "cockpituous@cockpit-project.org"
+          mkdir -p ~/.config/cockpit-dev
+          echo ${{ github.token }} >> ~/.config/cockpit-dev/github-token
+          eval $(ssh-agent)
+          ssh-add - <<< '${{ secrets.NODE_CACHE_DEPLOY_KEY }}'
```

Comment:
Redundant, the "force push" step does that.  I'd say, do the `./tools/node-modules push` in the second step (so that you keep all SSH operations in one step), or just merge them both.

Reply:
Uhmm there are two different deploy keys at play here, one for the `node-cache` repo and one for the pushing to the existing PR.

Reply:
Ah, I see, sorry!